src/components/look-controls.js

```diff
@@ -56,11 +57,21 @@ module.exports.Component = registerComponent('look-controls', {
     var data = this.data;
     if (!data.enabled) { return; }
     this.controls.standing = data.standing;
+    this.controls.userHeight = this.getUserHeight();
     this.controls.update();
     this.updateOrientation();
     this.updatePosition();
   },
 
+  /**
+   * Return user height to use for standing poses, where a device doesn't provide an offset.
+   */
+  getUserHeight: function () {
+    var headEl = this.el.sceneEl.camera.el;
```

Comment:
A scene could have multiple cameras. In this case we probably only care about the camera on the entity where this component is setup. I would use the public API this way:
````javascript 
var el = this.el;
var userHeight = el.hasAttribute('camera') && el.getAttribute('camera').userHeight || DEFAULT_CAMERA_HEIGHT;
return userHeight;
````

Reply:
Does this work in your case?

Reply:
I'm worried about the coupling of look-controls and the camera right now. Moving userHeight to look-controls is something we have discussed in the past but we can do it in a separate PR

Reply:
Yes, this works and looks better. Thank  you!